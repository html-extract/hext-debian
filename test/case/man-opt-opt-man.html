<a href="">
  <span class="-span1">
    <span class="--span1">
      <span class="---span1"></span>
    </span>
    <span class="--span2 NOPE">
      <b>nope</b>
    </span>
  </span>
</a>
