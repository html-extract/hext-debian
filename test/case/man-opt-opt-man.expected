{"first_opt_span":"-span1","secnd_opt_span":"--span1","trd_manda_span":"---span1"}
